tests/fixtures.ts

```diff
@@ -121,6 +157,28 @@ export const test = baseTest.extend<TestFixtures, WorkerFixtures>({
 
   mcpBrowser: ['chrome', { option: true }],
 
+  mcpExtension: [false, { option: true }],
+
+  mcpExtensionPage: async ({ mcpExtension, mcpHeadless }, use) => {
+    if (!mcpExtension)
+      return await use(undefined);
+    const pathToExtension = path.join(url.fileURLToPath(import.meta.url), '../../extension');
+    const context = await chromium.launchPersistentContext('', {
+      headless: mcpHeadless,
+      args: [
+        `--disable-extensions-except=${pathToExtension}`,
+        `--load-extension=${pathToExtension}`,
+        '--enable-features=AllowContentInitiatedDataUrlNavigations'
+      ],
+      channel: 'chromium',
+    });
+    const page = context.pages()[0];
+    page.on('dialog', () => {});
```

Comment:
why is this needed?

Reply:
Because we otherwise auto-dismiss the dialogs from the Playwright which launches the initial browser. Added a comment.